Misc.js

```diff
@@ -593,23 +594,32 @@ const LibraryNavButton = ({
   hasEn,
   withArrow,
   buttonStyle,
+  isMainMenu
```

Comment:
the `isMainMenu` case feels different enough to warrant it's own component. There are 4 if statements checking this value.